release notes/v0.37.0.md

```diff
@@ -0,0 +1,72 @@
+k6 v0.37.0 is here! 🎉 Mainly it contains fixes and ongoing efforts with refactoring, but it's just because we are preparing some incredible things in upcoming releases!
+
+## New Features!
+
+### Added support for local files to --log-output ([#2285](https://github.com/grafana/k6/pull/2285))
+
+Added file hook for a logger with the possibility to define a minimal reporting level.
```

Comment:
```suggestion
Added an option to output k6 logs to a file, on top of the already support stdout/stderr and loki instance.
```

"file hook" is some internal thing to logrus and will mean nothing to most users.